src/gui/sdlmain.cpp

```diff
@@ -5094,6 +5076,7 @@ int sdl_main(int argc, char* argv[])
 		// All subsystems' hotkeys need to be registered at this point
 		// to ensure their hotkeys appear in the graphical mapper.
 		MAPPER_BindKeys(sdl_sec);
+		GFX_RegenerateWindow(sdl_sec);
```

Comment:
And then I guess you hoisted this call out here so it only happens once on first launch?  Just trying to understand.

Reply:
yup.